cmd/orb.go

```diff
@@ -288,6 +330,70 @@ func releaseOrb(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func devLabel(label string) string {
+	// Ensure the `dev:` tag is prefixed to the label, no matter what
+	return fmt.Sprintf("dev:%s", strings.TrimPrefix(label, "dev:"))
+}
+
+func devOrb(cmd *cobra.Command, args []string) error {
+	ctx := context.Background()
+	response, err := api.OrbPublish(ctx, Logger, args[0], args[1], args[2], devLabel(args[3]))
+
+	if err != nil {
+		return err
+	}
+
+	if len(response.Errors) > 0 {
+		return response.ToError()
+	}
```

Comment:
Returning errors in two different ways (err and response.Errors) is not a great interface. Could this be handled in the `api` layer and returned as `err` ?